nifi-extension-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/main/java/org/apache/nifi/kafka/processors/PublishKafka.java

```diff
@@ -92,7 +92,8 @@
         + "The messages to send may be individual FlowFiles, may be delimited using a "
         + "user-specified delimiter (such as a new-line), or "
         + "may be record-oriented data that can be read by the configured Record Reader. "
-        + "The complementary NiFi processor for fetching messages is ConsumeKafka.")
+        + "The complementary NiFi processor for fetching messages is ConsumeKafka. "
+        + "To produce a kafka tombstone message while using PublishStrategy.USE_WRAPPER, simply set the value of a record to 'null'.")
```

Comment:
The previous implementation seems to suggest the expectation that `value=null` should only be sent when both conditions are true (zero byte, tombstone attribute).  But then it would be nice to be able to send `n` wrapper records, where some are tombstone records and some are not.  So this should be ok.

Reply:
Sorry, I don't understand this point! Is this related to the Wrapper change above (adding the tombstone attribute)?

Reply:
I was talking myself through the tradeoffs.  For USE_WRAPPER, this documentation seems fine.  No request here.